_posts/2019-07-1-foreman-community-newsletter---june-2019.md

```diff
@@ -0,0 +1,155 @@
+of these tasks is the "Did you know?" series of tweets that I will start from next week,
+so if you havn't yet followed @theforeman on twitter, you might want to do that now! Too
+much about my work, lets see what we missed in the recent 3 months.
+
+### 1.22.0 is out! (and Katello 3.12 is out too)!
+
+Work on [1.22][1_22] has been rolling on throughout the last couple of months, and it's
+finally out now! This release brings type support in Parameters,
+various compute resource libraries update, fixing the broken Google Compute
+Resource, Graphql API and Smart Proxy Exposed Capabilities and Settings. Read the
+[release notes][release_notes] to know more about the new features and *ton* of fixes done in this release.
+
+We had a test week kindly organized by [Lukas][lzap] in the
+simillar fashion as done in the previous release. We couldn't do this without
+the testing we get from the community, so thanks very much to you all!
+
+[1_22]: https://community.theforeman.org/t/foreman-1-22-0-rc2-has-been-released/14064
+[release_notes]: https://theforeman.org/manuals/1.22/index.html#Releasenotesfor1.22
+[lzap]: https://github.com/lzap
+
+As ever, many thanks to the contributors to these releases - coders, RC testers,
+documentation writers, and bug/feature reporters. Foreman wouldn't be what it
+is without you!
+
+### Conference roundup
+
+As ever, Foreman was present at Red Hat Summit this year,
+and as usual the Foreman/Pulp booth looked incredibly busy every time the expo
+area open. The talks by Tomer and Ewoud were well received. As Tomer
+mentioned in the community demo, a lot of questions were asked around Ansible
+integration, working of foreman with Ansible Tower, Remote execution and also
+acknowledged the improvements to content view publishing, better UX for repos
+and audit. Sounds like good news to us! All the work that we have been putting
+in is being noticed :)
+
+Not to forget, we had the very awesome yellow Foreman hats distributed at the Foreman
+booth. Thanks to Tomer and Ori for handling the design and shipment of the goodies.
+
+### Community Survey analysis is now out
+
+Thanks to everyone who filled out this year's community survey - we have now
+the analysis of the report that was prepared by Greg. We had 116 replies, which is a
+little down on last year, but still more than enough data to work with. As promised, Greg has
+written us a blog, which you can read [here][gregs_blog].
+
+[gregs_blog]: https://theforeman.org/2015/02/foreman-community-survey-results.html
+
+### [Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+All Foreman events are available via [the forum
+calendar](https://community.theforeman.org/calendar) - hit the Subscribe button
+(which will give you an ICS link). You can also use the "Add To Calendar" button
+on specific events if you don't wish to import the whole thing.
+
+* [Foreman Birthday Party](https://community.theforeman.org/t/foreman-birthday-party/13731)
```

Comment:
I would put this as it's own title and not part of the events since it is the birthday event and it's 10 years so it deserves a special section